### DISCUSSION TITLE
톰캣은 왜 9.0을 사용하였나요?

### DISCUSSION THREAD

**Discussion by hesoo98** [SIST1/TodayWithMyDoggy#125] [Q&A] [answered]
톰캣은 왜 9.0을 사용하였나요?

**Comment by tenta3802** [comment, marked as answer]
원래 톰캣 10을 사용하다가 서버 연동 오류로 인해 10을 사용했습니다. 검색 자료에 따르면 JavaEE와 JakartaEE. Tomcat 9와 Tomcat 10는 API 구현체가 완전히 다지만 Tomcat 9 이전까진 JavaEE 라는 동일한 스펙이기 때문에 문제 없이 사용했습니다
